### ISSUE TITLE
Bug located in ...

### ISSUE THREAD

**Issue by XIAO-GUO-commint** [ansys/pyansys-geometry#897]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

My question is How to enlarge an image and center it?

from ansys.geometry.core import launch_modeler_with_spaceclaim
modeler_discovery = launch_modeler_with_spaceclaim()
modeler_discovery.run_discovery_script_file( file_path=r'E:\RuanZhu-files\Optimization_Muti\parameters_modeling\1.scscript')
design=modeler_discovery.read_existing_design()
design.plot()

![image](https://github.com/ansys/pyansys-geometry/assets/83817101/4ae213a3-946b-41b9-a158-db13f449c873)

### 📝 Steps to reproduce

from ansys.geometry.core import launch_modeler_with_spaceclaim
modeler_discovery = launch_modeler_with_spaceclaim()
modeler_discovery.run_discovery_script_file( file_path=r'E:\RuanZhu-files\Optimization_Muti\parameters_modeling\1.scscript')
design=modeler_discovery.read_existing_design()
design.plot()

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

ansys2024r1

### 🐍 Which Python version are you using?

3.11

### 📦 Installed packages

```shell
Package                               Version
------------------------------------- ---------------
about-time                            4.2.1
aiohttp                               3.9.1
aiosignal                             1.3.1
alembic                               1.12.0
alive-progress                        3.1.4
altgraph                              0.17.4
ansys-api-dbu                         0.2.2
ansys-api-fluent                      0.3.18
ansys-api-geometry                    0.3.5
ansys-api-mapdl                       0.5.1
ansys-api-platform-instancemanagement 1.0.0
ansys-fluent-core                     0.18.0
ansys-fluent-visualization            0.8.0
ansys-geometry-core                   0.4.3
ansys-mapdl-core                      0.67.0
ansys-mapdl-reader                    0.52.20
ansys-math-core                       0.1.3
ansys-platform-instancemanagement     1.1.2
ansys-tools-path                      0.3.2
anyio                                 4.0.0
appdirs                               1.4.4
argon2-cffi                           23.1.0
argon2-cffi-bindings                  21.2.0
arrow                                 1.3.0
asttokens                             2.4.0
async-lru                             2.0.4
attrs                                 23.1.0
autograd                              1.6.2
Babel                                 2.13.0
backcall                              0.2.0
beartype                              0.16.4
beautifulsoup4                        4.12.2
bleach                                6.1.0
cachetools                            5.3.2
certifi                               2023.7.22
cffi                                  1.16.0
charset-normalizer                    3.3.0
click                                 8.1.7
cloudpickle                           3.0.0
clr-loader                            0.2.6
cma                                   3.2.2
colorama                              0.4.6
colorlog                              6.7.0
comm                                  0.1.4
contourpy                             1.1.1
cycler                                0.12.1
debugpy                               1.8.0
decorator                             5.1.1
defusedxml                            0.7.1
Deprecated                            1.2.14
dill                                  0.3.7
docker                                6.1.3
et-xmlfile                            1.1.0
executing                             2.0.0
fastjsonschema                        2.18.1
filelock                              3.12.4
fonttools                             4.43.1
fqdn                                  1.5.1
frozenlist                            1.4.0
fsspec                                2023.10.0
future                                0.18.3
geomdl                                5.3.1
google-api-core                       2.12.0
google-api-python-client              2.105.0
google-auth                           2.23.3
google-auth-httplib2                  0.1.1
googleapis-common-protos              1.61.0
grapheme                              0.6.0
greenlet                              3.0.0
grpcio                                1.59.0
grpcio-health-checking                1.48.2
h5py                                  3.10.0
httplib2                              0.22.0
idna                                  3.4
importlib-metadata                    6.8.0
ipykernel                             6.25.2
ipython                               8.16.1
ipython-genutils                      0.2.0
ipywidgets                            8.1.1
isoduration                           20.11.0
jedi                                  0.19.1
Jinja2                                3.1.2
joblib                                1.3.2
json5                                 0.9.14
jsonpointer                           2.4
jsonschema                            4.19.1
jsonschema-specifications             2023.7.1
jupyter                               1.0.0
jupyter_client                        8.4.0
jupyter-console                       6.6.3
jupyter_core                          5.4.0
jupyter-events                        0.8.0
jupyter-lsp                           2.2.0
jupyter_server                        2.8.0
jupyter_server_terminals              0.4.4
jupyterlab                            4.0.7
jupyterlab-pygments                   0.2.2
jupyterlab_server                     2.25.0
jupyterlab-widgets                    3.0.9
kiwisolver                            1.4.5
llvmlite                              0.41.1
lxml                                  4.9.3
Mako                                  1.2.4
MarkupSafe                            2.1.3
matplotlib                            3.8.0
matplotlib-inline                     0.1.6
mistune                               3.0.2
mpmath                                1.3.0
multidict                             6.0.4
multiprocess                          0.70.15
nbclient                              0.8.0
nbconvert                             7.9.2
nbformat                              5.9.2
nest-asyncio                          1.5.8
networkx                              3.2
notebook                              7.0.6
notebook_shim                         0.2.3
numba                                 0.58.1
numpy                                 1.26.1
openpyxl                              3.1.2
optuna                                3.4.0
overrides                             7.4.0
packaging                             23.2
pandas                                2.1.1
pandocfilters                         1.5.0
parso                                 0.8.3
pefile                                2023.2.7
pickleshare                           0.7.5
Pillow                                10.1.0
Pint                                  0.22
pip                                   23.3.1
platformdirs                          3.11.0
plotly                                5.18.0
pooch                                 1.7.0
prometheus-client                     0.17.1
prompt-toolkit                        3.0.39
protobuf                              3.20.3
psutil                                5.9.6
pure-eval                             0.2.2
pyansys-tools-versioning              0.5.0
pyasn1                                0.5.0
pyasn1-modules                        0.3.0
pycparser                             2.21
Pygments                              2.16.1
pyiges                                0.3.1
pyinstaller                           6.3.0
pyinstaller-hooks-contrib             2023.11
pymoo                                 0.6.0.1
pyparsing                             3.1.1
PyQt5                                 5.15.9
pyqt5-plugins                         5.15.9.2.3
PyQt5-Qt5                             5.15.2
PyQt5-sip                             12.13.0
pyqt5-tools                           5.15.9.3.3
PyQtWebEngine                         5.15.6
PyQtWebEngine-Qt5                     5.15.2
PySide6                               6.6.0
PySide6-Addons                        6.6.0
PySide6-Essentials                    6.6.0
python-dateutil                       2.8.2
python-dotenv                         1.0.0
python-json-logger                    2.0.7
pythonnet                             3.0.3
pytz                                  2023.3.post1
pyvista                               0.42.3
pyvistaqt                             0.11.0
pywin32                               306
pywin32-ctypes                        0.2.2
pywinpty                              2.0.12
PyYAML                                6.0.1
pyzmq                                 25.1.1
qt5-applications                      5.15.2.2.3
qt5-tools                             5.15.2.1.3
qtconsole                             5.4.4
QtPy                                  2.4.0
referencing                           0.30.2
requests                              2.31.0
rfc3339-validator                     0.1.4
rfc3986-validator                     0.1.1
rpds-py                               0.10.6
rsa                                   4.9
SALib                                 1.4.7
scikit-learn                          1.3.1
scipy                                 1.11.3
scooby                                0.9.2
seaborn                               0.13.0
Send2Trash                            1.8.2
setuptools                            68.2.2
shap                                  0.43.0
shiboken6                             6.6.0
six                                   1.16.0
slicer                                0.0.7
sniffio                               1.3.0
soupsieve                             2.5
SQLAlchemy                            2.0.22
stack-data                            0.6.3
sympy                                 1.12
tenacity                              8.2.3
terminado                             0.17.1
threadpoolctl                         3.2.0
tinycss2                              1.2.1
torch                                 2.1.0
torchvision                           0.16.0
tornado                               6.3.3
tqdm                                  4.66.1
traitlets                             5.11.2
trame                                 3.3.0
trame-client                          2.13.0
trame-server                          2.12.1
types-python-dateutil                 2.8.19.14
typing_extensions                     4.8.0
tzdata                                2023.3
uri-template                          1.3.0
uritemplate                           4.1.1
urllib3                               2.0.7
vtk                                   9.3.20230807rc0
wcwidth                               0.2.8
webcolors                             1.13
webencodings                          0.5.1
websocket-client                      1.6.4
wheel                                 0.41.2
widgetsnbextension                    4.0.9
wrapt                                 1.15.0
wslink                                1.12.4
xgboost                               2.0.0
yarl                                  1.9.4
zipp                                  3.17.0
```

**Comment by RobPasMue** [comment]
@AlejandroFernandezLuces please answer this user.

**Comment by XIAO-GUO-commint** [comment]
Thank you very much! Both of your methods are effective, but it seems that the second method cannot open the PyVista interface in Jupyter Notebook.

**Comment by AlejandroFernandezLuces** [comment]
We are currently working on the Jupyter Notebook visualization, since it has recently changed in PyVista and we are still adapting to it. Sorry for the inconvenience and we'll let you know once this is implemented.

**Comment by AlejandroFernandezLuces** [comment]
> Thank you very much! Both of your methods are effective, but it seems that the second method cannot open the PyVista interface in Jupyter Notebook.

This was just fixed in PR #904 , to use it you can either use the dev version of the package or wait for the next release of the package. Closing this issue as it is solved, let me know if it works for you if you try it out 🙂
